soc/riscv/litex-vexriscv/Kconfig.soc

```diff
@@ -6,3 +6,6 @@ config SOC_RISCV32_LITEX_VEXRISCV
 	select RISCV
 	select ATOMIC_OPERATIONS_C
 	select INCLUDE_RESET_VECTOR
+
+config LITEX_CSR_DATA_WIDTH
+	int "Select Control/Status register width"
```

Comment:
I think we need a default value for this, otherwise no sample will build unless this is explicitly set

Reply:
I added default value of 32 to _boards/riscv/litex_vexriscv/litex_vexriscv_defconfig_

Reply:
Actually, I declared a default value here and moved the whole symbol into an `if` block. Otherwise, it would be visible even when this SoC isn't selected, making it a choice symbol, which can't have a default value.